docker/mongodb-kubernetes-tests/tests/conftest.py

```diff
@@ -836,6 +839,29 @@ def _install_multi_cluster_operator(
     # The Operator will be installed from the following repo, so adding it first
     helm_repo_add("mongodb", "https://mongodb.github.io/helm-charts")
 
+    # helm_chart_path not being passed would mean we are on evg env and would like to
+    # install helm chart from OCI registry.
+    if not helm_chart_path:
+        # login to the OCI container registry
+        registry, repository, region = oci_chart_info()
+        try:
+            oci_helm_registry_login(registry, region)
+        except Exception as e:
+            raise e
+
+        # figure out the registry URI, based on dev/staging scenario
+        chart_uri = f"oci://{registry}/{repository}"
+
+        helm_chart_path = chart_uri
+
+    if not custom_operator_version and helm_chart_path not in (MCK_HELM_CHART, LEGACY_OPERATOR_CHART):
+        # custom_operator_version is not set and chart is also neither MCK nor MEKO would mean we are trying to install
+        # the operator from OCI registry. The version based on build scenario (dev/staging) is set in `OPERATOR_VERSION` env var.
+        non_semver_custom_operator_version = os.environ.get(OPERATOR_VERSION_ENV_VAR_NAME)
+        # when we publish the helm chart we append `0.0.0+` in the chart version, details are
+        # here https://docs.google.com/document/d/1eJ8iKsI0libbpcJakGjxcPfbrTn8lmcZDbQH1UqMR_g/edit?tab=t.gg5ble8qlesq
+        custom_operator_version = f"0.0.0+{non_semver_custom_operator_version}"
```

Comment:
this will not work when testing against released version which uses semantic version i.e. when we run e2e smoke tests after publishing images to production